fhir-path/src/main/java/org/linuxforhealth/fhir/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -450,6 +483,36 @@ private Collection<FHIRPathNode> is(Collection<ExpressionContext> arguments) {
             FHIRPathNode node = getSingleton(currentContext);
             return type.isAssignableFrom(node.type()) ? SINGLETON_TRUE : SINGLETON_FALSE;
         }
+        
+        /**
+         * https://hl7.org/fhirpath/N1/#is-type-specifier
+         * https://hl7.org/fhirpath/N1/#istype-type-specifier
+         * This method returns true if the type of the left operand is the type specified in the second operand, 
+         * If the input value is not of the type, this method returns false
+         * @param arguments the parse tree
+         * @return the result of evaluation as a non-null, collection of FHIRPath nodes(The boolean value wrapped by this FHIRPathBooleanValue node)
+         */
+        private Collection<FHIRPathNode> isTypeEqual(Collection<ExpressionContext> arguments) {
+            if (arguments.size() != 1) {
+                throw unexpectedNumberOfArguments(arguments.size(), "is");
```

Comment:
```suggestion
                throw unexpectedNumberOfArguments(arguments.size(), "isTypeEqual");
```

Reply:
This is fixed as suggested. replaced "is" with "isTypeEqual".